staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go

```diff
@@ -112,9 +115,9 @@ func NewCmdCreateDeployment(f cmdutil.Factory, ioStreams genericiooptions.IOStre
 	cmdutil.AddApplyAnnotationFlags(cmd)
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddDryRunFlag(cmd)
-	cmd.Flags().StringSliceVar(&o.Images, "image", o.Images, "Image names to run.")
+	cmd.Flags().StringSliceVar(&o.Images, "image", o.Images, "Image names to run. A deployment can have multiple images set for multi-container pod. It is to be noted that command can't be mentioned along with list of images. ")
```

Comment:
What do you mean by `It is to be noted that command can't be mentioned along with list of images.`?

Reply:
It's been a while since I made this commit but what i can recall is that you can't write multiple container name etc.

Reply:
i guess this makes it more confusing . so I'll remove the line.